coursier.bzl

```diff
@@ -440,7 +440,8 @@ def remove_auth_from_url(url):
     host = url_parts[0]
     if "@" not in host:
         return url
-    userless_host = host[host.find("@") + 1:]
+    last_index=host.rfind("@", 0, None)
```

Comment:
FYI there are some split_url() tests that have weird urls that this probably won't work for. Not sure if those are real cases somebody has, or not. 

https://github.com/bazelbuild/rules_jvm_external/blob/master/tests/unit/coursier_test.bzl#L208

Reply:
I think it's not realistic to have a host with @ 
My commit should fix those 2 cases:
username@whatever.com:password@host
username@whatever.com@itLikes@icanputanotheronehere@:password_with_@_why_not:host

Reply:
I agree, I don't think having that @ char in the path, in unencoded form, is even legal. But just FYI someone thought it was a valid use case.